Index: src/com/company/ReverseBubbleSort.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.company;\n\npublic class ReverseBubbleSort2 {\n    private static final int[] NUMBERS = {12,324,112,15,4,134,345,3,2,4,1,5,9};\n\n    private static int[] sortedArray(int[] numbers) {\n        boolean isSorted = false;\n        while (!isSorted) {\n            isSorted = true;\n            for (int i = 1; i < numbers.length; i++) {\n                int current = numbers[i];\n                int previous = numbers[i - 1];\n\n                if (current > previous) {\n                    int temp = numbers[i];\n                    numbers[i] = numbers[i - 1];\n                    numbers[i - 1] = temp;\n                    isSorted = false;\n                }\n            }\n        }\n        return numbers;\n    }\n\n    private static void print(int[] sortedArray) {\n        for (int number : sortedArray) {\n            System.out.print(number + \" \");\n        }\n    }\n\n    public static void main(String[] args) {\n        print(sortedArray(NUMBERS));\n    }\n}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/com/company/ReverseBubbleSort.java	(revision 065e27092963b316d97c7c3dca17cc43ab9b32ba)
+++ src/com/company/ReverseBubbleSort.java	(date 1661503940352)
@@ -1,6 +1,6 @@
 package com.company;
 
-public class ReverseBubbleSort2 {
+public class ReverseBubbleSort {
     private static final int[] NUMBERS = {12,324,112,15,4,134,345,3,2,4,1,5,9};
 
     private static int[] sortedArray(int[] numbers) {
